src/main/java/io/cryostat/agent/ConfigModule.java

```diff
@@ -642,4 +719,28 @@ public static URIRange fromString(String s) {
             return SITE_LOCAL;
         }
     }
+
+    public static class ByteBuffer {
+        private final byte[] buf;
+
+        public ByteBuffer(int len) {
+            this.buf = new byte[len];
+        }
+
+        public ByteBuffer(byte[] s) {
+            this.buf = Arrays.copyOf(s, s.length);
```

Comment:
Should be careful here - if we're just copying the array in, then we are able to control the new copy and clear it as needed, but the original one is still potentially floating around out there containing the contents indefinitely. Wherever this constructor is invoked should take care to clear the original array too once the ByteBuffer has been constructed.